### DISCUSSION TITLE
How to get ViewContext given Editor and WindowContext?

### DISCUSSION THREAD

**Discussion by thataboy** [zed-industries/zed#19088] [Feature Requests]
Can someone tell me how I can get `ViewContext<'_,  Editor>` given `Editor` and `WindowContext`?

Thanks in advance

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
